run/healthchecks_startup_probe_grpc/main.tf

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Enable Cloud Run API
+resource "google_project_service" "cloudrun_api" {
```

Comment:
JLYK: API is enabled at https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf#L62